src/parser/wsdl/operation.js

```diff
@@ -42,6 +42,7 @@ class Operation extends WSDLElement {
       case 'operation': // soap:operation
         this.soapAction = child.$soapAction || '';
         this.style = child.$style || '';
+        this.soapActionRequired = child.$soapActionRequired || 'false';
```

Comment:
Would it be better to convert the value to boolean?

Reply:
Incorporated the comment to change this.soapActionRequired to boolean